src/babel/transformation/templates/create-class.js

```diff
@@ -1,7 +1,8 @@
 (function() {
   function defineProperties(target, props) {
-    for (var key in props) {
-      var prop = props[key];
+    var keys = Object.getOwnPropertyNames(props).concat(Object.getOwnPropertySymbols(props));
```

Comment:
This is problematic, since behavior differs in polyfilled and non-polyfilled environment. Only solution for this I can think of is adding a check if Object.getOwnPropertySymbols is defined.